src/main/java/seedu/address/model/module/Module.java

```diff
@@ -0,0 +1,92 @@
+package seedu.address.model.module;
+
+import java.time.LocalDateTime;
+import java.util.Objects;
+
+public class Module {
+    private Title title;
+    private AssignmentList assignments;
+    private Exam exam;
```

Comment:
Do you think we need an exam list as well since a module can take in multiple exams at the same time.